armi/utils/directoryChangers.py

```diff
@@ -133,6 +134,19 @@ def _retrieveEntireFolder(self):
     def _transferFiles(initialPath, destinationPath, fileList):
         """
         Transfer files into or out of the directory.
+        
+        Parameters
+        ----------
+        initialPath : str
+            Path to the folder to find files in.
+        destinationPath: str
+            Path to the folder to move file to.
+        fileList : list of str or list of tuple
+            File names to move from initial to destination. If this is a 
+            simple list of strings, the files will be transferred. Alternatively
+            tuples of (initialName, finalName) are allowed if you want the file
+            renamed during transit. In the non-tuple option, globs/wildcards
+            are allowed.
```

Comment:
Wildly pedantic, but what we really have here is a `List[Union[Tuple[str, str], str]]`. Is "list of str and/or tuple" valid numpydoc?

Reply:
I think it's at least close https://numpydoc.readthedocs.io/en/latest/format.html#sections